chart/templates/dag-processor/dag-processor-deployment.yaml

```diff
@@ -229,6 +231,10 @@ spec:
         {{- if .Values.dagProcessor.extraContainers }}
           {{- tpl (toYaml .Values.dagProcessor.extraContainers) . | nindent 8 }}
         {{- end }}
+            - name: tmp
+              mountPath: /tmp 
+            - name: opt
+              mountPath: /opt/airflow                      
```

Comment:
Doesn't it make more sense to define the `/opt/airflow` mountPath based on the `airflowHome` value and have the name more generic like `airflow-home`?

Reply:
yeah that probably makes more sense

Reply:
So this would be changed in the dag-processor, flower, scheduler, webserver, worker deployments. 

There are some yaml tamplates though where /opt/airflow is hardcoded already so I believe I was following that.  For example if you look in basic_template.yaml 

Im not certain if all files in the "dag-processor, flower, scheduler, webserver, worker deployments" pods are only created in the airflow_home directory, or if both should be mounted, /opt/airflow and whatever airflowhome is set to.  

Let me know if that makes sense.

Reply:
Just pushed the change you suggested 👍